code/modules/spacepods/spacepod.dm

```diff
@@ -233,12 +233,19 @@
 	if(!health)
 		spawn(0)
 			message_to_riders("<span class='userdanger'>Critical damage to the vessel detected, core explosion imminent!</span>")
-			for(var/i = 10, i >= 0; --i)
-				message_to_riders("<span class='warning'>[i]</span>")
-				if(i == 0)
-					explosion(loc, 2, 4, 8)
-					qdel(src)
+			for(var/i in 1 to 3)
+				var/count = 1
```

Comment:
... what is the point of this
use `i` instead

Reply:
There was a point to this and I seem to have done it completely the wrong way. It's supposed to count down from 3 to 1 not up. Using `i` would make it count up. Which is why I did this. Then made  it count up anyway... 😂